components/server/src/workspace/workspace-starter.ts

```diff
@@ -1341,6 +1341,14 @@ export class WorkspaceStarter {
         contextEnv.setValue(JSON.stringify(workspace.context));
         envvars.push(contextEnv);
 
+        const info = this.config.workspaceClasses.find((cls) => cls.id === instance.workspaceClass);
+        if (!!info) {
```

Comment:
why do we need the `!!`?

Reply:
I'd agree here, `if` is already checking truthyness / falsyness of the expression [[1](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/if...else)], so we are basically doing the boolean conversion twice.

Reply:
@mustard-mh did you see these comments?

Reply:
> @mustard-mh did you see these comments?

@andreafalzetti  Thanks for your reminder 🙏 . I have saw this, but want left it to webApp team to answer it 😂, I use `!!` because I see it exists in some typescript code

<img width="339" alt="image" src="https://user-images.githubusercontent.com/20944364/187369093-cffdc68d-d5d3-4d74-a9eb-6393dd623de1.png">